<launch>

  <node pkg="lab_ros_aruco" type="lab_ros_aruco_node" name="AruCoMakerNode" output="screen">
   
    <param name="marker_dictonary" value="ARUCO_MIP_36h12"/>
    <param name="marker_size" value="0.045"/>

    <remap from="image" to="/usb_cam/image_raw"/>
    <remap from="camera_info" to="/usb_cam/camera_info"/>

    <remap from="markers" to="/markersAruco"/>
  </node>

</launch>
